### ISSUE TITLE
Changing method signature in other class in not recognized in working class in typed GDScript

### ISSUE THREAD

**Issue by pgoral** [godotengine/godot#28685]
**Godot version:**
84401e8
**Issue description:**
I have two classes A and B. B calls method foo from A class. Method has one parameter.
```
class_name A
func foo(one)

class_name B
func bar():
	a.foo(1)
```
Now I am changing foo signature to foo(one, two) and I have no error in my B class.
When I reopen B class file I see the error "Too few arguments..."

This could be duplicate of #28002

**Comment by pgoral** [comment]
Guys I was investigating this, and I did a quick fix but I don't feel it's the right way. So I need some guidance.
I am sending patch witch changes I have made.
[bug.txt](https://github.com/godotengine/godot/files/3154777/bug.txt)

**Comment by akien-mga** [comment]
Fixed by #28766.
